static/app/components/dropdownMenuItem.tsx

```diff
@@ -199,6 +199,7 @@ const BaseDropdownMenuItem: React.ForwardRefRenderFunction<HTMLLIElement, Props>
 
   return (
     <MenuListItem
+      aria-haspopup={isSubmenuTrigger}
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-haspopup

Used to indicate that this `menuitem` is going to open a submenu

Reply:
Nice!